src/Widget/Attribute/ModelAttributes.php

```diff
@@ -13,7 +14,7 @@ trait ModelAttributes
     private array $attributes = [];
     private string $attribute = '';
     private string $charset = 'UTF-8';
-    private string $id = '';
+    private ?string $id = '';
```

Comment:
```suggestion
    private ?string $id = null;
```

Reply:
Because we are going to eliminate the id by default, it is necessary that the widgets have an id.

Reply:
All widgets need id?

Reply:
If they all have id, some use `HtmlForm::getInputId()`, which is generated by the model, others use `Html::generateId()`.